src/Bicep.Core/FileSystem/PathHelper.cs

```diff
@@ -137,6 +139,27 @@ public static Uri FilePathToFileUrl(string filePath)
             return uriBuilder.Uri;
         }
 
+        public static string? TryGetUsingPath(UsingDeclarationSyntax? usingDeclarationSyntax, out DiagnosticBuilder.ErrorBuilderDelegate? failureBuilder)
+        {
+            var pathSyntax = usingDeclarationSyntax?.TryGetPath();
+            if (pathSyntax == null)
+            {
+                failureBuilder = null;
```

Comment:
Ideally, either the return value should be non-null or the value of `failureBuilder` should be.  I think in this case you want something similar to https://github.com/Azure/bicep/blob/57c2d2207bfca63a42e7daa0715e717f19355703/src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs#L604-L607